cmds/fmap/fmap.go

```diff
@@ -89,8 +115,18 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	switch {
+	// Optionally print checksum.
+	case *checksum != "":
+		checksum, err := f.Checksum(r, hashFuncs[*checksum])
```

Comment:
so if you did not call the New() but only had the pointer then this could be 
f().Checksum(etc. etc.)